### ISSUE TITLE
TypeError: Cannot read property 'off' of undefined

### ISSUE THREAD

**Issue by aleqcz** [alladdin/node-lox-mqtt-gateway#5]
Hello Alladdin,
I have updated to 0.3.1 and executed the gateway for the first time. Previously undefined control Presence doesn't block it anymore, but about a second-two later it crashed with following messages:

```
...
2017-06-23T10:55:36.675Z - debug: WebSocketAPI - received update event:  uuid=0d745096-0003-028e-fffffae984b19f22, event=0
2017-06-23T10:55:36.675Z - debug: MQTT Adaptor - for mqtt:  topic=miniserver/global/state, data={"operatingMode":"Pátek","sunrise":290,"sunset":1260,"notifications":0}
2017-06-23T10:55:36.675Z - debug: WebSocketAPI - received update event:  uuid=0d84b37a-027a-3493-ffffabe90c3f4505, event=0
2017-06-23T10:55:36.676Z - debug: MQTT Adaptor - for mqtt:  topic=miniserver/global/state, data={"operatingMode":"Pátek","sunrise":290,"sunset":1260,"notifications":0,"modifications":0}
2017-06-23T10:55:36.676Z - debug: WebSocketAPI - received update event:  uuid=0f78179d-019c-0224-ffff504f941059fb, event=0
2017-06-23T10:55:36.676Z - debug: WebSocketAPI - received update event:  uuid=0f78179d-019c-0225-ffff504f941059fb, event=0
2017-06-23T10:55:36.677Z - debug: WebSocketAPI - received update event:  uuid=0f9667cb-0015-068c-fffffae984b19f22, event=0
/home/node/node_modules/node-lox-mqtt-gateway/node_modules/node-lox-structure-file/lib/Structure/Control/InfoOnlyDigital.js:14
        'text': this.details.text[value],
                                 ^

TypeError: Cannot read property 'off' of undefined
    at InfoOnlyDigital.get_state (/home/node/node_modules/node-lox-mqtt-gateway/node_modules/node-lox-structure-file/lib/Structure/Control/InfoOnlyDigital.js:14:34)
    at InfoOnlyDigital.<anonymous> (/home/node/node_modules/node-lox-mqtt-gateway/lib/Adaptor.js:94:104)
    at emitOne (events.js:96:13)
    at InfoOnlyDigital.emit (events.js:188:7)
    at UUID.<anonymous> (/home/node/node_modules/node-lox-mqtt-gateway/node_modules/node-lox-structure-file/lib/Structure/Control.js:31:22)
    at emitThree (events.js:116:13)
    at UUID.emit (events.js:194:7)
    at UUID.set_value (/home/node/node_modules/node-lox-mqtt-gateway/node_modules/node-lox-structure-file/lib/Structure/UUID.js:22:10)
    at UUID.Base.set_value_for_uuid (/home/node/node_modules/node-lox-mqtt-gateway/node_modules/node-lox-structure-file/lib/Structure/Base.js:48:23)
    at ListByKey.<anonymous> (/home/node/node_modules/node-lox-mqtt-gateway/node_modules/node-lox-structure-file/lib/Structure/Base.js:53:46)

```

Thanks for looking. Best regards,
Aleq

**Comment by alladdin** [comment]
I hope that it's fixed in new released version 0.3.2

**Comment by aleqcz** [comment]
Working! Thank you very much.
I was able to launch and have the bridge running for the first time.

Have a nice weekend,
  Aleq
